tests/setup.js

```diff
@@ -60,14 +53,13 @@ beforeAll(() => {
 });
 
 beforeEach(() => {
-    logs = [];
     warns = [];
     errors = [];
 });
 
 afterEach(() => {
-    if (logs.length > 0 || warns.length > 0 || errors.length > 0) {
-        const message = 'Unexpected console logs' + logs + warns + errors;
+    if (warns.length > 0 || errors.length > 0) {
```

Comment:
Minor change to match how redux handled log statements in its tests. I personally think this is nicer since I tend to use `console.log` while debugging tests.